setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = CMinx
+version = 1.0
```

Comment:
Won't this decouple version information from git tags?

Reply:
whoops. I was using that for testing and never pushed the real fix.